packages/x-date-pickers/src/DesktopDateTimePicker/DesktopDateTimePickerLayout.tsx

```diff
@@ -40,7 +40,7 @@ function DesktopDateTimePickerLayout<
       {isLandscape ? shortcuts : toolbar}
       {isLandscape ? toolbar : shortcuts}
       <PickersLayoutContentWrapper
-        className={pickersLayoutClasses.contentWrapper}
+        className={clsx(pickersLayoutClasses.contentWrapper, classes?.contentWrapper)}
```

Comment:
The addition of the test allowed me to discover this "bug"—the `slotProps.layout.classes` were not used in the case of `DesktopDateTimePicker` and `DesktopDateTimeRangePicker`. 🙈